frontend/admin/src/pages/purchase/PurchaseDetail.tsx

```diff
@@ -38,6 +38,7 @@ const PurchaseDetailInternal = (props: any) => {
     });
     const toast = useToast();
     const history = useHistory();
+    const refresh = () => setTimeout(() => history.go(0), 1500);
```

Comment:
Why do we need a timeout to refresh the page?

Reply:
problem is that the toast cannot be displayed after refresh. scripts are reloaded too.
Therefore, the workaround is to wait 1.5s, so that the toast can be seen before refresh for now

Reply:
Ok, but then the implementation is wrong. We were able to navigate between different pages using `history.push` without toasts getting lost.

Reply:
Reverted refresh for the moment, no refresh is being made after status change in current branch